app/views/partials/_header.html.haml

```diff
@@ -0,0 +1,13 @@
+%nav.navbar.navbar-expand-lg.fixed-top
+  %a.navbar-brand{ href: '/' }
+    = image_tag('abuelometro_logo.png', alt: 'Abuelometro logo', class: 'navbar-logo')
+    Abuelometro
+  %button.navbar-toggler{ type: 'button', data: { toggle: 'collapse', target: '#navbarToggler'}, aria: { controls: 'navbarToggler', expanded: 'false', label: 'Toggle navigation'} }
+    %span.navbar-toggler-icon
+  #navbarToggler.collapse.navbar-collapse
+    %ul.navbar-nav
+      %li.nav-item
+        %a.nav-link{ href: '/admin' } Iniciar sesión
+      %li.nav-item
+        %a.nav-link{ href: '/about-us' }
```

Comment:
If you are going to use haml I think you should take advantage of all of its helpers. for all the `%a` you can use `= link 'Text you want to insert', 'URL it will take you to'`

Reply:
Sounds good to me.